cmd/main.go

```diff
@@ -266,15 +271,15 @@ func setupKymaReconciler(mgr ctrl.Manager, remoteClientCache *remote.ClientCache
 	}
 }
 
-func createSkrWebhookManager(mgr ctrl.Manager, flagVar *FlagVar) (*watcher.SKRWebhookManifestManager, error) {
+func createSkrWebhookManager(mgr ctrl.Manager, flagVar *FlagVar) (*watcher.WatcherManifestManager, error) {
 	caCertificateCache := watcher.NewCACertificateCache(flagVar.caCertCacheTTL)
-	return watcher.NewSKRWebhookManifestManager(mgr.GetConfig(), mgr.GetScheme(), caCertificateCache,
-		watcher.SkrWebhookManagerConfig{
-			SKRWatcherPath:         flagVar.skrWatcherPath,
-			SkrWatcherImage:        flagVar.skrWatcherImage,
-			SkrWebhookCPULimits:    flagVar.skrWebhookCPULimits,
-			SkrWebhookMemoryLimits: flagVar.skrWebhookMemoryLimits,
-			RemoteSyncNamespace:    flagVar.remoteSyncNamespace,
+	return watcher.NewWatcherManifestManager(mgr.GetConfig(), mgr.GetScheme(), caCertificateCache,
+		watcher.WatcherManagerConfig{
+			WatcherResourcesPath: flagVar.watcherResourcesPath,
+			WatcherImage:         fmt.Sprintf("%s/%s", flagVar.watcherRegistry, flagVar.watcherImage),
```

Comment:
can you rename this `WatcherManagerConfig.WatcherImage` to something which can distinguish with `flagVar.watcherImage`